src/ocamlorg_package/lib/ocamlorg_package.ml

```diff
@@ -301,15 +301,26 @@ let documentation_page ~kind t path =
       Some Documentation.{ content; toc; module_path }
   | Error _ -> Lwt.return None
 
-let readme_file ~kind t =
-  let open Lwt.Syntax in
-  let+ doc = documentation_page ~kind t "README.md.html" in
-  match doc with None -> None | Some { content; _ } -> Some content
+  let maybe_file filename ~kind t =
+    let open Lwt.Syntax in
+    let+ doc = documentation_page ~kind t filename in
+    match doc with None -> None | Some { content; _ } -> Some content
+
+let readme_file ~kind = maybe_file "README.md.html" ~kind
 
-let license_file ~kind t =
+let license_file ~kind = maybe_file "LICENSE.md.html" ~kind
+
+let changes_file ~kind t =
   let open Lwt.Syntax in
-  let+ doc = documentation_page ~kind t "LICENSE.md.html" in
-  match doc with None -> None | Some { content; _ } -> Some content
+  let* md = maybe_file "CHANGES.md.html" ~kind t in
+  match md with
+  | Some doc -> Lwt.return_some (true, doc)
+  | None ->
+    let+ txt = maybe_file "CHANGES.html" ~kind t in
```

Comment:
Perhaps not useful if we only do this for the current function, but we could have a `maybe_files` function that takes a list of file names and will return the content of the first one that matches. So the `changes_files` would be implemented as:
```ocaml
let changes_file ~kind t =
  maybe_files [ "CHANGES.md.html"; "CHANGES.html"; "CHANGELOG.md.html" ]
```